GAE/test/org/akvo/flow/rest/ImageUploadRestServiceTest.java

```diff
@@ -79,10 +79,8 @@ void testUploadImageWhenFormNotFound() throws IOException {
         Long formInstanceId = surveyInstance.getKey().getId();
 
         ImageUploadRestService service = new ImageUploadRestService();
-        ImageUploadRestService.Response response = service.uploadImage(123L, formInstanceId, null);
 
-        assertEquals(400, response.getCode());
-        assertEquals("Form not found", response.getMessage());
+        assertThrows(Exception.class, () -> service.uploadImage(123L, formInstanceId, null), "Form not found");
```

Comment:
Is it possible for this exception to be more specific? i.e. the `ReponseStatusException`?